project-resources/MEMBER_TRAVEL_FUND.md

```diff
@@ -11,9 +11,14 @@ and spread knowledge about and support the Javascript ecosystem and the Foundati
 
 ## Requirements
 
-* Candidates must be an active participant of one of the projects of the OpenJS Foundation.
```

Comment:
why did you change the bullet types in the dock? Is there a linter issue? If we are going to do maintenance type work like this it is probably good to do it as a separate PR as it is creating a lot of noise for this change.

Reply:
I havent changed the bullet types i just checked my local file and there is no sign of * but will be careful going further